rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityFilter.vue

```diff
@@ -56,17 +56,18 @@
       :query="modelValue"
       :has-query="hasQuery"
       :event-bus="eventBus"
-      @select_filter="selectFilter($event)"
       data-test-id="saved-filters"
+      @select_filter="selectFilter($event)"
     ></saved-filters>
     <modal
       id="activityFilter"
+      ref="filterModal"
```

Comment:
please remove the refs that aren't being used, it's preferable to use data-testids for tests whenever it's possible